sonic-xcvrd/scripts/xcvrd

```diff
@@ -348,107 +314,26 @@ class dom_info_update_task:
         self.task_stopping_event.set()
         self.task_timer.join()
 
-#=============================== Main ================================
+# Returns path to port config file
+def get_path_to_port_config_file():
```

Comment:
Should this function become a method of DaemonBase class so it can be shared among daemons, if needed?

Reply:
Given that this function will only be used by xcvrd, I think we can put this function to DaemonXcvrd rather than DaemonBase.

Reply:
OK. We can consider moving it in the future if it ever needs to be shared.